.github/workflows/devnet-sync.yml

```diff
@@ -2,24 +2,15 @@ name: Devnet sync
 
 on:
```

Comment:
We should change this condition to `push` for the sake of testing so we can see that the job works before we merge this.